code401challenges/arrayReverse/array-reverse.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+public class ArrayReverse {
```

Comment:
You file should have the same name as this class, so your file should be called ArrayReverse.java